### PR TITLE
added managed prometheus alert manager config for  memcached and haproxy

### PR DESCRIPTION
**Pull request by rameshv29**
### What does this PR do?

added managed prometheus alert manager config for  memcached and haproxy

### More

- [ x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/observability/adot-amp-grafana-for-haproxy/main.tf [hunk 1]
```diff
@@ -144,3 +144,77 @@ resource "grafana_dashboard" "haproxy_dashboards" {
...
+    expr: sum by (server) (rate(haproxy_server_http_responses_total{code="5xx"}[1m])) / sum by (server) (rate(haproxy_server_http_responses_total[1m]) * 100) > 5
+    for: 1m
+    labels:
+      severity: critical
+    annotations:
+      summary: HAProxy high HTTP 5xx error rate server (instance {{ $labels.instance }})
+      description: "Too many HTTP requests with status 5xx (> 5%) on server {{ $labels.server }}\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
+  EOF
+}
+#Configure AWS Managed Prometheus alert manager
```
Comment (bryantbiggs) [inline examples/observability/adot-amp-grafana-for-haproxy/main.tf:210]: 1 space between resource stanzas please

**Comment by bryantbiggs** [comment]
Thanks @rameshv29 - if you could fix the line space and re-run `pre-commit run -a` we should be good. Please note - I updated your branch with `main` so please make sure you pull that down locally before updating. There are some changes from a prior PR that will be updated when you run the pre-commit command

**Comment by rameshv29** [comment]
Thank you @bryantbiggs ! I have added a line space and ran pre-commit without any issues. Can you please review and merge these changes?

**Review by bryantbiggs — APPROVED** [review]
Thank you @rameshv29 🎉

**Pull request merged**